docs/src/modules/components/TopLayoutCompany.js

```diff
@@ -10,6 +10,7 @@ import MarkdownElement from './MarkdownElement';
 const styles = (theme) => ({
   root: {
     flex: '1 0 100%',
+    marginTop: 30,
```

Comment:
```suggestion
    // Adding top buffer because of the v5 banner
    marginTop: 30,
```

Reply:
So we need to remove it once we remove the banner? Can we link these two locations so that it's obvious that both need to change?

Reply:
This is targetting master, are we ever going to change the docs for v4?

Reply:
We are going to merge at some point, yes.

Reply:
I will open a follow up PR to link all updates that we did after the first PR.

Reply:
Fixed with https://github.com/mui-org/material-ui/pull/27309